problem installing your program

I tried to install your program, and I followed the instructions exactly, but I get an error message. I did the commands:

git clone https://github.com/Voudy/image_morphing.git
cd image_morphing
mkdir build
cd build
cmake ..

Everything works properly until I type in the last command `make`. When I do that, I receive the following error message:

`[ 50%] Building CXX object CMakeFiles/image_morphing.dir/main.cpp.o
/home/jr/image_morphing/main.cpp: In function ‘void draw_point(cv::Mat&, const Point2f&, const Scalar&)’:
/home/jr/image_morphing/main.cpp:17:31: error: ‘CV_FILLED’ was not declared in this scope
     circle(img, fp, 2, color, CV_FILLED, CV_AA, 0);
                               ^~~~~~~~~
/home/jr/image_morphing/main.cpp:17:31: note: suggested alternative: ‘__FILE’
     circle(img, fp, 2, color, CV_FILLED, CV_AA, 0);
                               ^~~~~~~~~
                               __FILE
/home/jr/image_morphing/main.cpp:17:42: error: ‘CV_AA’ was not declared in this scope
     circle(img, fp, 2, color, CV_FILLED, CV_AA, 0);
                                          ^~~~~
/home/jr/image_morphing/main.cpp:17:42: note: suggested alternative: ‘CV_8U’
     circle(img, fp, 2, color, CV_FILLED, CV_AA, 0);
                                          ^~~~~
                                          CV_8U
/home/jr/image_morphing/main.cpp: In function ‘void draw_triangulation(cv::Mat&, cv::Subdiv2D&, const Scalar&)’:
/home/jr/image_morphing/main.cpp:32:56: error: ‘CV_AA’ was not declared in this scope
             line(img, pt[0], pt[1], delaunay_color, 1, CV_AA, 0);
                                                        ^~~~~
/home/jr/image_morphing/main.cpp:32:56: note: suggested alternative: ‘CV_8U’
             line(img, pt[0], pt[1], delaunay_color, 1, CV_AA, 0);
                                                        ^~~~~
                                                        CV_8U
CMakeFiles/image_morphing.dir/build.make:62: recipe for target 'CMakeFiles/image_morphing.dir/main.cpp.o' failed
make[2]: *** [CMakeFiles/image_morphing.dir/main.cpp.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/image_morphing.dir/all' failed
make[1]: *** [CMakeFiles/image_morphing.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
`

Why is this occurring, and how do I think this problem? I am using Linux Mint.

No, still doesn't work. I made the change to line 17, but I am still getting the same error message shown below when I get to the make step:
`/home/jr/image_morphing/main.cpp: In function ‘void draw_point(cv::Mat&, const Point2f&, const Scalar&)’:
/home/jr/image_morphing/main.cpp:17:43: error: ‘CV_AA’ was not declared in this scope
     circle(img, fp, 2, color, cv::FILLED, CV_AA, 0);
                                           ^~~~~
/home/jr/image_morphing/main.cpp:17:43: note: suggested alternative: ‘CV_8U’
     circle(img, fp, 2, color, cv::FILLED, CV_AA, 0);
                                           ^~~~~
                                           CV_8U
/home/jr/image_morphing/main.cpp: In function ‘void draw_triangulation(cv::Mat&, cv::Subdiv2D&, const Scalar&)’:
/home/jr/image_morphing/main.cpp:32:56: error: ‘CV_AA’ was not declared in this scope
             line(img, pt[0], pt[1], delaunay_color, 1, CV_AA, 0);
                                                        ^~~~~
/home/jr/image_morphing/main.cpp:32:56: note: suggested alternative: ‘CV_8U’
             line(img, pt[0], pt[1], delaunay_color, 1, CV_AA, 0);
                                                        ^~~~~
                                                        CV_8U
CMakeFiles/image_morphing.dir/build.make:62: recipe for target 'CMakeFiles/image_morphing.dir/main.cpp.o' failed
make[2]: *** [CMakeFiles/image_morphing.dir/main.cpp.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/image_morphing.dir/all' failed
make[1]: *** [CMakeFiles/image_morphing.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
`

I also tried chaning CV_AA to cv::AA on line 17 too, but that doesn't fix things either.

--- a/main.cpp
+++ b/main.cpp
@@ -14,7 +14,7 @@ using namespace std;
 using namespace cv;
 
 static void draw_point(Mat & img, const Point2f &fp, const Scalar &color) {
-    circle(img, fp, 2, color, CV_FILLED, CV_AA, 0);
+    circle(img, fp, 2, color, FILLED, LINE_AA, 0);
 }
 
 static void draw_triangulation(Mat & img, Subdiv2D & subdiv, const Scalar &delaunay_color) {

